quantum-viz/tests/test_qiskit_generate_results.py

```diff
@@ -0,0 +1,19 @@
+from quantum_viz.qiskit_parser import qiskit2json
+from quantum_viz.utils import display
+
+from .conftest import qc_to_path
+
+
+def test_generate_results(
```

Comment:
this test is overwriting the content of the tests file with whatever `qiskit2json` generates. I'd rather not have this and just have a hardcoded json in the tests so we can detect changes accordingly.
 What I would do is to remove this "test" and just commit it as a script, i.e. rename the file to not have `test` in it, and just make the body the content of this method.
I would also then add a comment at the top of `test_qiskit_parser.py` explaining how to use this file to re-generate the results if necessary.

Reply:
Done in https://github.com/jond01/quantum-viz.js/pull/7